spotify/serializers.py

```diff
@@ -0,0 +1,12 @@
+# Takes all python related code and translates Room to json response 
+
+from rest_framework import serializers 
+from .models import SpotifyToken
+
+# Take SpotifyToken object and serialize it 
+class userSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = SpotifyToken
+        fields = ('id', 'user', 'created_at', 'refresh_token', 'access_token',
+        'expires_in', 'token_type')
```

Comment:
Just asking out of curious, this probably is the case, but this doesn't respond with the HTTP 200 response correct?

Reply:
If that's not the case, where does this JSON go to? Again, out of curiosity and learning.

Reply:
> Just asking out of curious, this probably is the case, but this doesn't respond with the HTTP 200 response correct?

correct


> If that's not the case, where does this JSON go to? Again, out of curiosity and learning.

this will only be viewable from the admin page and not viewable by users of our website